components/script/dom/htmlareaelement.rs

```diff
@@ -35,8 +189,28 @@ impl HTMLAreaElement {
                prefix: Option<DOMString>,
                document: &Document) -> Root<HTMLAreaElement> {
         Node::reflect_node(box HTMLAreaElement::new_inherited(local_name, prefix, document),
-                           document,
-                           HTMLAreaElementBinding::Wrap)
+               document,
+               HTMLAreaElementBinding::Wrap)
+    }
+
+    #[warn(dead_code)]
+    fn get_shape_from_coords(&self) -> Option<Area> {
+       let elem = self.upcast::<Element>();
+       let shp: Shape;
+       let shape = elem.get_string_attribute(&LocalName::from("shape"));
+
+       match shape.as_ref() {
```

Comment:
Do you want us to add circle, rectangle and polygon to the static_atoms list? I didn't quite get the comment.